projects/imagenet/run.py

```diff
@@ -49,4 +46,4 @@
     config = process_args(args, config)
 
     if config is not None:
-        imagenet_tune.run(config)
+        vernon.run_with_raytune(config)
```

Comment:
Missing handling of `single_instance` argument created by `RAY_PARSER`. 
See my previous comment